### DISCUSSION TITLE
News

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#519] [Announcements]
This thread will be used to announce about

* new MTEX versions
* upcoming MTEX workshops
* MTEX related publications

This allows you to kept informed by clicking the **subscribe** button.

Ralf.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very proud to announce MTEX 5.5. which includes major bug fixes in the [volume computation function](https://mtex-toolbox.github.io/ODF.volume.html) and many other small improvements. As new features it offers now a much more [comfortable way to analyze low angle grain boundaries ](https://mtex-toolbox.github.io/SubGrainBoundaries.html) and
introduces [orientational embeddings](https://mtex-toolbox.github.io/OrientationEmbeddings.html) which allows to work with orientation data as with ordinary tensors without the need to care about symmetry.

Please check the [changelog](https://mtex-toolbox.github.io/changelog.html) for more details. 

We hope you enjoy MTEX,

The MTEX team.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very happy to announce the [MTEX workshop 2021](https://www-user.tu-chemnitz.de/~rahi/mtexWorkshop21/index.php) which will be organized as an online workshop from March 8th to March 17th 2021. 

The purpose of the workshop is twofold. The first week is an MTEX training course that also provides basic knowledge on crystallography and texture analysis. We believe that MTEX provides an efficient way to grain better understanding on these subjects. After successfully attending  this course you will get a certificate which counts for 4 ETCS points. 

The second week it planed as MTEX user meeting. All current and future MTEX users are invited to share specific use cases, tips and tricks and discuss recent and upcoming developments in MTEX. The schedule includes introductory presentations of invited speakers to general use cases of MTEX as well as 20 minutes talks by the participants on specific applications.

I would be very happy to have nice online workshop with as many discussions as we had in the last years.
Hope to see you soon.

Ralf Hielscher and Rüdiger Kilian.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very proud to announce [MTEX 5.6](https://github.com/mtex-toolbox/mtex/releases/download/mtex-5.6.0/mtex-5.6.0.zip) which features a major revision of the parent grain reconstruction functionality. Most importantly, it is now much more easier and user friendly to reconstruct parent grain structures in various setting. See [here](https://mtex-toolbox.github.io/MaParentGrainReconstruction.html) and [here](https://mtex-toolbox.github.io/TiBetaReconstruction.html) for examples and have a look at the [changelog](https://mtex-toolbox.github.io/changelog.html).

We also greatly improved the compatibility of MTEX with older Matlab versions. 

We hope you enjoy MTEX and would be happy to welcome you at the [MTEX workshop 2021](https://www-user.tu-chemnitz.de/~rahi/mtexWorkshop21/index.php). 

The [MTEX team](https://mtex-toolbox.github.io/people).

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very happy to announce the MTEX workshop 2022 (https://mtex-toolbox.github.io/workshops) which will be organized as an online workshop from March 21st to March 30th 2020.

The purpose of the workshop is twofold. The first week is an MTEX training course with focus on data cleaning, interpretation and visualization. Participants can send in their data, that will analyzed during the exercises. The course also provides basic knowledge on crystallography and texture analysis, as we believe that MTEX significantly helps understanding these subjects. After successfully attending this course you will get a certificate which counts for 5 ETCS points.

The second week it planed as an MTEX user meeting. All current and future MTEX users are invited to share specific use cases, tips and tricks and discuss recent and upcoming developments in MTEX. The schedule includes introductory presentations of invited speakers to general use cases of MTEX as well as 20 minutes talks by the participants on specific applications.

I would be very happy to have nice online workshop with as many discussions as we had in the last years. Hope to see you soon.

Ralf Hielscher, Rüdiger Kilian, Luiz Morales and Frank Niessen.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very proud to announce [MTEX 5.8](https://mtex-toolbox.github.io/download) which implements the new [variant graph based parent grain reconstruction algorithm](https://arxiv.org/abs/2201.02103).

We hope you enjoy MTEX and would be happy to welcome you at the [MTEX workshop 2022](https://mtex-toolbox.github.io/workshops).

The [MTEX team](https://mtex-toolbox.github.io/people).

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are proud to announce MTEX 5.8.2 which contains many bug fixes and some minor improvements.

The MTEX team.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very happy to announce the MTEX workshop 2023 (https://mtex-toolbox.github.io/workshop23) from March 6th to March 15th 2023. Although the workshop will allow for remote participation and all lectures will be streamed and recorded, we strongly encourage everybody to join in presence in order to enjoy the discussions, coffee breaks and excursion in real life.

The purpose of the workshop is twofold. The first week is an MTEX training course with focus on data cleaning, interpretation and visualization. Participants can send in their data, that will analyzed during the exercises. The course also provides basic knowledge on crystallography and texture analysis, as we believe that MTEX significantly helps understanding these subjects. After successfully attending this course you will get a certificate which counts for 5 ETCS points.

The second week it planed as an MTEX user meeting. All current and future MTEX users are invited to share specific use cases, tips and tricks and discuss recent and upcoming developments in MTEX. The schedule includes introductory presentations of invited speakers to general use cases of MTEX as well as 20 minutes talks by the participants on specific applications. Another topic will be the substantial changes in MTEX 6.0 which will be released next year.

We are looking forward to a wonderful first workshop in presence after to years in the online word. Hope to see you soon.

Ralf Hielscher, Rüdiger Kilian, Luiz Morales and Frank Niessen.

**Reply by AdamLeaghj** [reply]
Hi Ralf,

I'm from Canada and really want to join this one virtually. What is the registration fee for the virtual form? Thanks

**Reply by ralfHielscher** [reply]
Yes, the registration fee is the same for in presence and virtual participation. We did so, in order to stimulate the people to attend the workshop in presence.

All the best and a happy new year,

Ralf.

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very proud to announce [MTEX 5.10](https://github.com/mtex-toolbox/mtex/releases/download/mtex-5.10.0/mtex-5.10.0.zip) which implements the weighted Burgers vector method from J. Wheeler and many other minor improvements, speedups and bug fixes. For a full list of changes check out the [changelog](https://mtex-toolbox.github.io/changelog#1).

We hope you enjoy MTEX.

The MTEX team.

**Comment by windsornewstodayca** [comment]
Windsor News Today

**[Breaking News Headlines](https://windsornewstoday.ca/)**

Stay updated with latest Breaking News Headlines and today's news in Windsor. Stay informed with up-to-the-minute updates.

Discover the current weather in vibrant Windsor. Stay informed about weather conditions and forecasts for your day.

Contact info

(519) 258-3434

📷 windsoressex@blackburnnews.com

[https://windsornewstoday.ca/](https://windsornewstoday.ca/)

**Comment by ralfHielscher** [comment]
Dear MTEX community,

we are very proud to announce [MTEX 6.1](https://github.com/mtex-toolbox/mtex/releases/download/mtex-6.1.0/mtex-6.1.0.zip) which solidifies all the new features of MTEX 6.0 and contains a massive amount of bug fixes and speed improvements. In particular we hope we made the installation process on Windows and Mac computers more straight forward.

We hope you enjoy it,

Your MTEX team.

**Reply by NovakEvana** [reply]
Thank you, Ralf and the MTEX team!
